### DISCUSSION TITLE
uplink supply and demand

### DISCUSSION THREAD

**Discussion by deltanedas** [space-wizards/space-station-14#17975] [Feature Request]
so people dont winge about fishops too op
what if you buy too many fish and the price gets hiked until its not possible to buy anymore
this would also make other items cheaper maybe to encourage more diverse strategies

**Comment by Androclast** [comment]
sec metagaming what contraband is in play by anomalous pricing on certain items

**Reply by deltanedas** [reply]
per-uplink

**Comment by hord-brayden** [comment]
Would shared supply and demand be interesting? Meaning, all of the syndicate share a pool of items, if you will. There's x amount of fish, x amount of emag, c4, implants etc. It could be interesting to force people into new uplink items because their usual go-to's have been taken. And it may force people to try and arm up earlier on, so as to ensure they procure the items they'll need for their objectives. Just peanuting

**Reply by deltanedas** [reply]
frontier station liberation vend pro
